src/sass/components/_position.scss

```diff
@@ -1,7 +1,59 @@
+// Position Item
+
+@include media-object('.position-item', '.position-item__avatar', '.position-item__content');
+
 .position-item {
-  .position-source {
-    color: rgba(171,171,171, 1);
-    font-size: 0.9em;
-    font-weight: 200;
-  }
+  position: relative;
```

Comment:
Sure. This class has the responsibility right now of handling the box model, position, and how it looks. Would it not be better to have those broken apart and possibly change the class name for something more generic that could be reusable?